src/equations/numerical_fluxes.jl

```diff
@@ -219,13 +219,13 @@ struct FluxHLL{MinMaxSpeed}
     min_max_speed::MinMaxSpeed
 end
 
-FluxHLL() = FluxHLL(min_max_speed_naive)
+FluxHLL() = FluxHLL(min_max_speed)
```

Comment:
This is a breaking change.

Reply:
Reverted in f1e7f5beb4cc8d630d5199cfb444e98a23330fe1